pkg/controller/clone-controller.go

```diff
@@ -171,11 +170,19 @@ func (c *CloneController) processPodItem(pod *v1.Pod) error {
 	// see if pvc's pod phase anno needs to be added/updated
 	phase := string(pod.Status.Phase)
 	if !checkIfAnnoExists(pvc, AnnPodPhase, phase) {
-		pvc, err = setPVCAnnotation(c.clientset, pvc, AnnCloneOf, "true")
+		pvc, err = setPVCAnnotation(c.clientset, pvc, AnnPodPhase, phase)
```

Comment:
why is the cloner doing this update? The importer controller will do it in its onw `ProcessPodItem`

Reply:
Don't we want the pod phase written to the cloning PVC's too?